packages/gatsby/src/state-machines/__tests__/develop.ts

```diff
@@ -103,7 +103,7 @@ describe(`the top-level develop state machine`, () => {
     expect(services.reloadData).toHaveBeenCalled()
   })
 
-  it(`queues a node mutation during query running`, () => {
+  it(`runs a node mutation during query running and marks node as dirty`, () => {
```

Comment:
Maybe group tests related to query resolvers into a separate `describe` section?